Widgets/ArenaWidget.lua

```diff
@@ -81,6 +83,26 @@ local function GetUnitArenaNumber(guid)
   --return ArenaID[guid]
 end
 
+local function GetFrameSortNumber(unitId)
+  local fs = FrameSortApi and FrameSortApi.v2
```

Comment:
Do you know what is best practice to check if an addon is loaded? Like this using a global variable or using IsAddonLoaded? Both work.

Also, we could move this to OnEnabled, but then it probably would break if FrameSort is disabled while the UI is not reloaded. Not sure though if that ever happens.

Reply:
I could add a check with IsAddonLoaded but I think I'd end up checking for the global variable anyway.